src/main/java/daronek6/cinema/entities/Ticket.java

```diff
@@ -0,0 +1,33 @@
+package daronek6.cinema.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Data
+@NoArgsConstructor
+@RequiredArgsConstructor
+public class Ticket {
+
+    private @Id @GeneratedValue Long id;
+    @NonNull
+    private TicketType type;
+    @NonNull
+    private double price;
+    @NonNull
+    private LocalDateTime priceStartTime;
+    private LocalDateTime priceEndTime;
```

Comment:
Tu chyba próbowałeś zrobić coś więcej niż przewiduje zadanie, fajnie, że miałeś ambicję :slightly_smiling_face: 
Rozumiem, że gdzieś po drodze przestałeś i to zostało?

Reply:
Tak, założyłem, że klient będzie chciał mieć możliwość zrobienia raportu sprzedaży biletów z danego okresu, a cena biletów w międzyczasie może się zmienić. Czas zakończenia ceny biletu pokazuje do kiedy ta cena była aktualna, jeśli ten czas nie jest ustalony (null) to znaczy, że aktualna cena biletu tego rodzaju znajduje się w tym rekordzie.